### PR TITLE
Fix MPT CI

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Hello, let's just fix this ....

### CODE REVIEW

File: tests/models/mpt/test_modeling_mpt.py [hunk 1]
```diff
@@ -507,7 +507,7 @@ def test_model_logits(self):
 
         outputs = model(dummy_input, output_hidden_states=True)
 
-        expected_slice = torch.Tensor([-0.2559, -0.2197, -0.2480]).to(torch_device, torch.bfloat16)
+        expected_slice = torch.Tensor([-0.2539, -0.2178, -0.1953]).to(torch_device, torch.bfloat16)
```
Comment (ydshieh) [inline tests/models/mpt/test_modeling_mpt.py:510]: I am not sure why the diff is so large ...

**Review by ArthurZucker — APPROVED** [review]
Fine with me,  I think this was pytorch / kernel related so should be good cc @younesbelkada

**Comment by ydshieh** [comment]
Merge now. Leave comment if any @younesbelkada when you are back 🙏

**Comment by younesbelkada** [comment]
Looks great @ydshieh , thanks!

**Comment by ydshieh** [comment]
Hello

**Pull request merged**